target/linux/ath79/dts/qca9558_belkin_f9j1108-v2.dts

```diff
@@ -0,0 +1,228 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca955x.dtsi"
+
+/ {
+	model = "Belkin F9J1108v2 (AC1750 DB Wi-Fi)";
```

Comment:
Nitpick, but I would add a space before the v2 here as well.

Reply:
The label doesn't have a space, so was trying to keep it the same

Reply:
But we are treating it as separated v2 in the image/file name and in the DEVICE_MODEL. Actually, this is the only case where it's not separated.